src/util.cpp

```diff
@@ -747,7 +748,12 @@ double double_safe_multiplication(double fValue, double fmultiplicator)
 
 void runCommand(std::string strCommand)
 {
+    if (strCommand.empty()) return;
+#ifndef WIN32
```

Comment:
Topic for a future PR, do we need this if we are not shipping 32-bit windows anymore?

Reply:
We do not ship **binaries** anymore.  But we haven't completely dropped any support for WIN32 (yet)

Reply:
`WIN32` is a generic precompiler macro for covering both 32-bit and 64-bit windows compiler toolchains, so it isn't specifically targeting 32-bit windows, but rather ALL windows.